src/main/app/src/styles.less

```diff
@@ -317,3 +317,7 @@ mat-paginator[sticky] {
 .tooltip-list {
   white-space: pre;
 }
+
+.mat-snackbar {
+  transform: translateY(4em) !important;
```

Comment:
!important maakt alles stuk. :-(

Alleen in noodgevallen gebruiken we dat. Enne translate? Was gewoon een position relative met een offset geen optie? Of een of andere margin?

Zet hem anders maar op mij, dan kijk ik wel even of ik daar wat nette CSS voor kan vinden.

Reply:
Margin, position en transform worden allemaal overridden door de `mat-mdc-snack-bar-container`-klasse die inline ingevoegd wordt, dus !important is hier echt nodig.

Ik gebruik translate omdat die semantisch logischer is dan margin. Margin zorgt er voor dat de bounding box van het element opgevuld wordt om het element als content te 'verschuiven'. Position zou hier wel kunnen, maar dan moeten er drie stijlattributen (`position` (is nu `static`), `bottom` (is nu `0`) en `top` (is nu niet geset)) aangepast worden:
```
.mat-snackbar {
  position: fixed !important;
  bottom: auto !important;
  top: 4em;
}
```
Ik vind persoonlijk een translate hier logischer, omdat de flow van het element niet veranderd, alleen de positie waar hij gerenderd wordt door de browser - maar voor de oplossing met position is ook wat te zeggen omdat het element sowieso al bovenop de flow van de rest van de pagina gerenderd wordt.

Reply:
Misschien moet je nog even kijken naar hoe CSS selectors werken, als je zegt dat een !important "echt nodig is".

Reply:
Bedankt voor de goede aanwijzing! Ik heb me er verder in verdiept - dit kan opgelost worden door de specificiteit van de selector aan te passen. Opgelost in opvolgende commit.